DemoApp-Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/InterstitialAdActivity.kt

```diff
@@ -0,0 +1,73 @@
+package com.applovin.enterprise.apps.demoapp.ads
+
+import android.os.Bundle
+import android.os.Handler
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.applovin.enterprise.apps.demoapp.kotlin.R
+import com.applovin.mediation.MaxAd
+import com.applovin.mediation.MaxAdListener
+import com.applovin.mediation.ads.MaxInterstitialAd
+
+/**
+ * [android.app.Activity] used to show AppLovin MAX interstitial ads.
+ * <p>
+ * Created by Harry Arakkal on 2019-09-17.
+ */
+class InterstitialAdActivity : AppCompatActivity(),
+        MaxAdListener
+{
+    private var interstitialAd: MaxInterstitialAd? = null
```

Comment:
You can use [`lateinit val`](https://kotlinlang.org/docs/reference/properties.html) to avoid forced unwrapping.

Same for RVs and Banners

Reply:
I was using `lateinit` before, but the docs use an optional like this. 

@thomasmso is it fine to use `lateinit` to clean up this code?

Reply:
I think it's fine and more clear - docs should be updated if anything.

Reply:
We did that in Swift too.